iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/analyzer/Scope.java

```diff
@@ -50,6 +54,7 @@ public class Scope {
   private final RelationId relationId;
   private final RelationType relation;
   private final Map<String, WithQuery> namedQueries;
+  private final List<Identifier> tables;
```

Comment:
what's this field used for? we need some comments to explain.

Reply:
done